README.md

```diff
@@ -1,4 +1,39 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go` https://github.com/golang/go/wiki/SettingGOPATH
+
+## get code and deps
+
+This will get the sources into default `GOPATH` which is `~/go`.
```

Comment:
This is technically correct, a better way would be something like
`This will get the sources into your GOPATH which by default is ~/go`

Reply:
ok